articles/overview/understanding-quantum-computing.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Understanding quantum computing
+description: What are quantum computers and how do they use the principles of quantum mechanics?
+author: bradben
+ms.author:  bradben
+ms.date: 5/5/2020
+ms.topic: overview
+uid: microsoft.quantum.overview.understanding
+---
+
+# Understanding quantum computing
+
+Quantum computing uses the principles of quantum mechanics to process information. Because of this, quantum computing requires a different approach than classical computing.  One example of this difference is the processor used in quantum computers.  Where classical computers use familiar silicon-based chips, quantum computers use quantum material such as atoms, ions, photons, or electrons.  
+
+The quantum material behaves according to the laws of quantum mechanics, leveraging concepts such as probabilistic computation, superposition, and entanglement. These concepts provide the basis for quantum algorithms that harness the power of quantum computing to solve complex problems. This article describes some of the essential concepts of quantum mechanics on which quantum computing is based.
+
+## A bird’s-eye view of quantum mechanics
+
+Quantum mechanics, also called quantum theory, is a branch of physics that deals with particles at the atomic and subatomic levels. At the quantum level, however, many of the laws of mechanics you take for granted don’t apply. Superposition, quantum measurement, and entanglement are three phenomena that are central to quantum computing.  
```

Comment:
I'd argue that interference is a more fundamental phenomena than measurement, as quantum interference determines the result of a measurement. "Collapse" as a result of measurement is an easy escape from explaining what's really happening fundamentally. I would suggest a rewrite of the superposition, measurement and entanglement sections, perhaps if not this time round, definitely soon.

Reply:
Absolutely, let's revisit this after \\build